Readme.md

```diff
@@ -5,117 +5,34 @@
 [![License](https://img.shields.io/github/license/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/blob/main/LICENSE)
 [![docker](https://badgen.net/docker/pulls/aquasec/tracee)](https://hub.docker.com/r/aquasec/tracee)
 
-# Tracee: Linux Runtime Security and Forensics using eBPF
+# Tracee: Runtime Security and Forensics using eBPF
 
-Tracee is a Runtime Security and forensics tool for Linux. It is using Linux eBPF technology to trace your system and applications at runtime, and analyze collected events to detect suspicious behavioral patterns.
+Tracee is a Runtime Security and forensics tool for Linux. It is using Linux eBPF technology to trace your system and applications at runtime, and analyze collected events to detect suspicious behavioral patterns. It is delivered as a Docker image that monitors the OS and detects suspicious behavior based on a pre-defined set of behavioral patterns.
 
-Tracee is delivered as a Docker image that once run, will start to monitor the OS and detect suspicious behavior based on a pre-defined set of behavioral patterns.
+## Documentation
 
-Tracee is composed of the following sub-projects:
-- [Tracee-eBPF](tracee-ebpf) - Linux Tracing and Forensics using eBPF
-- [Tracee-Rules](tracee-rules) - Runtime Security Detection Engine
-- [libbpgo](libbpfgo) - Go library for eBPF programming using Linux's [libbpf](https://github.com/libbpf/libbpf)
-
-## Getting started
-
-### Prerequisites
+The full documentation of Tracee is available at [https://aquasecurity.github.io/tracee](https://aquasecurity.github.io/tracee).
```

Comment:
Do I understand correctly that this link won't work until after this merges? Because it's broken right now.

Reply:
yes you understand correctly